phoenicis-javafx/src/main/java/org/phoenicis/javafx/settings/JavaFxSetting.java

```diff
@@ -20,8 +20,10 @@
 
 public enum JavaFxSetting {
     THEME("application.theme"), SCALE("application.scale"), VIEW_SOURCE("application.viewsource"), WINDOW_HEIGHT(
-            "application.windowHeight"), WINDOW_WIDTH(
-                    "application.windowWidth"), WINDOW_MAXIMIZED("application.windowMaximized");
+            "application.windowHeight"), WINDOW_WIDTH("application.windowWidth"), WINDOW_MAXIMIZED(
```

Comment:
Can we please fix the format of this enum definition?
I believe an enum to be better readable, if one value is defined in each row.